From: Alex Gonzalez <alex.gonzalez@digi.com>
Date: Tue, 3 Apr 2012 13:52:50 +0200
Subject: [PATCH] mfw_v4lsec_def_sizes

Change the default dimensions to the ones supported by the mt9v111 driver.

Signed-off-by: Alex Gonzalez <alex.gonzalez@digi.com>
---
 src/misc/v4l_source/src/mfw_gst_v4lsrc.c | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/misc/v4l_source/src/mfw_gst_v4lsrc.c b/src/misc/v4l_source/src/mfw_gst_v4lsrc.c
index d1057e3..968deb7 100755
--- a/src/misc/v4l_source/src/mfw_gst_v4lsrc.c
+++ b/src/misc/v4l_source/src/mfw_gst_v4lsrc.c
@@ -1304,8 +1304,8 @@ IMPORTANT NOTES:    None
 =============================================================================*/
 static void mfw_gst_v4lsrc_init(MFWGstV4LSrc * v4l_src, MFWGstV4LSrcClass * klass)
 {
-    v4l_src->capture_width=176;
-    v4l_src->capture_height=144;
+    v4l_src->capture_width=640;
+    v4l_src->capture_height=480;
     v4l_src->fps_n=30;
     v4l_src->fps_d=1;
     v4l_src->fd_v4l=-1;
@@ -1315,12 +1315,12 @@ static void mfw_gst_v4lsrc_init(MFWGstV4LSrc * v4l_src, MFWGstV4LSrcClass * klas
     v4l_src->crop_pixel=0;
     v4l_src->rotate=0;
     v4l_src->preview=FALSE;
-    v4l_src->preview_width=160;
-    v4l_src->preview_height=128;
+    v4l_src->preview_width=640;
+    v4l_src->preview_height=480;
     v4l_src->preview_top=0;
     v4l_src->preview_left=0;
-    v4l_src->sensor_width=1280;
-    v4l_src->sensor_height=1024;
+    v4l_src->sensor_width=640;
+    v4l_src->sensor_height=480;
     v4l_src->capture_mode = 0;
     v4l_src->bg = FALSE;
 #ifdef MX27
@@ -1372,14 +1372,14 @@ static void mfw_gst_v4lsrc_class_init(MFWGstV4LSrcClass * klass)
 				    g_param_spec_int("capture-width",
 						     "capture_width",
 						     "gets the width of the image to be captured",
-						     16, 1768, 176,
+						     16, 1768, 640,
 						     G_PARAM_READWRITE));
 
     g_object_class_install_property(gobject_class, MFW_V4L_SRC_HEIGHT,
 				    g_param_spec_int("capture-height",
 						     "capture_height",
 						     "gets the height of the image to be captured",
-						     16, 1168, 144,
+						     16, 1168, 480,
 						     G_PARAM_READWRITE));
 
     g_object_class_install_property(gobject_class, MFW_V4L_SRC_PREVIEW_WIDTH,
@@ -1387,7 +1387,7 @@ static void mfw_gst_v4lsrc_class_init(MFWGstV4LSrcClass * klass)
 						     "preview_width",
 						     "gets the width of the image to be displayed for preview. \n"
                              "\t\t\tNote:property is valid only when preview property is enabled",
-						     16, 1768, 176,
+						     16, 1768, 640,
 						     G_PARAM_READWRITE));
 
     g_object_class_install_property(gobject_class, MFW_V4L_SRC_PREVIEW_HEIGHT,
@@ -1395,7 +1395,7 @@ static void mfw_gst_v4lsrc_class_init(MFWGstV4LSrcClass * klass)
 						     "preview_height",
 						     "gets the height of the image to be displayed for preview. \n"
                              "\t\t\tNote:property is valid only when preview property is enabled",
-						     16, 1168, 144,
+						     16, 1168, 480,
 						     G_PARAM_READWRITE));
 
     g_object_class_install_property(gobject_class, MFW_V4L_SRC_PREVIEW_TOP,
@@ -1471,14 +1471,14 @@ static void mfw_gst_v4lsrc_class_init(MFWGstV4LSrcClass * klass)
 						       "sensor_width",
 						       "gets the width of the sensor", 
 						       16,
-						       G_MAXINT, 1280,
+						       G_MAXINT, 640,
 						       G_PARAM_READWRITE));
      g_object_class_install_property(gobject_class, MFW_V4L_SRC_SENSOR_HEIGHT,
 				    g_param_spec_int("sensor-height",
 						       "sensor_height",
 						       "gets the height of the sensor", 
 						       16,
-						       G_MAXINT, 1024,
+						       G_MAXINT, 480,
 						       G_PARAM_READWRITE));
 
      g_object_class_install_property(gobject_class, MFW_V4L_SRC_CAPTURE_MODE,
